src/components/WorkPoolAverageLateTime.vue

```diff
@@ -1,39 +1,41 @@
 <template>
-  <span v-if="averageLateness" class="work-pool-average-late-time">
-    ({{ secondsToApproximateString(averageLateness) }} avg.)
+  <span v-if="lateness" class="work-pool-average-late-time">
+    ({{ secondsToApproximateString(lateness) }} avg.)
   </span>
 </template>
 
 <script lang="ts" setup>
-  import { useSubscription } from '@prefecthq/vue-compositions'
-  import { computed } from 'vue'
-  import { useInterval, useWorkspaceApi } from '@/compositions'
+  import merge from 'lodash.merge'
+  import { toRef } from 'vue'
+  import { useFlowRunsAverageLateness, useInterval } from '@/compositions'
   import { FlowRunsFilter, WorkPool } from '@/models'
+  import { Getter, MaybeGetter } from '@/types/reactivity'
   import { secondsToApproximateString } from '@/utilities'
 
   const props = defineProps<{
     workPool: WorkPool,
-    filter?: FlowRunsFilter,
+    filter?: MaybeGetter<FlowRunsFilter>,
   }>()
 
-  const api = useWorkspaceApi()
+  const flowRunsFilter = toRef(props.filter)
   const options = useInterval()
 
-  const lateFlowRunsFilter = computed<FlowRunsFilter>(() => ({
-    ...props.filter,
-    workPools: {
-      name: [props.workPool.name],
-    },
-    flowRuns: {
-      ...props.filter?.flowRuns,
-      state: {
-        name: ['Late'],
+  const lateFlowRunsFilter: Getter<FlowRunsFilter> = () => {
+    const filter: FlowRunsFilter = {
+      workPools: {
+        name: [props.workPool.name],
       },
-    },
-  }))
+      flowRuns: {
+        state: {
+          name: ['Late'],
```

Comment:
Am I correct in assuming the API accepts and we have use cases for passing flow run states that _aren't_ late to the average lateness composition? If not, it might make sense to bake this into the composition

Reply:
Hmm good catch. I'm actually not sure this is correct at all. But I'll check on it

Reply:
This shouldn't even have the late state filter. So removed that. Thanks for calling that out!